tests/test_charset.py

```diff
@@ -41,16 +41,22 @@ def testEncodings(self):
             try:
                 test_string.encode(x[1])
             except LookupError as e:
-                self.fail(msg="DICOM term '%s' does not have a valid python encoding (was '%s')" % (x[0], x[1]))
+                found = "(was '%s')" % x[1]
+                term = "Term '%s'" % x[0]
+                message = "%s has invalid python encoding %s" % (found, term)
+                self.fail(msg=message)
 
     def testNestedCharacterSets(self):
         """charset: can read and decode SQ with different encodings........."""
         ds = dicomio.read_file(sq_encoding_file)
         ds.decode()
-        # These datasets inside of the SQ cannot be decoded with default_encoding
-        # OR UTF-8 (the parent dataset's encoding). Instead, we make sure that it
-        # is decoded using the (0008,0005) tag of the dataset
-        expected = u'\uff94\uff8f\uff80\uff9e^\uff80\uff9b\uff73=\u5c71\u7530^\u592a\u90ce=\u3084\u307e\u3060^\u305f\u308d\u3046'
+
+        # These datasets inside of the SQ cannot be decoded with
+        # default_encoding OR UTF-8 (the parent dataset's encoding).
+        # Instead, we make sure that it is decoded using the
+        # (0008,0005) tag of the dataset
+
+        expected = u'\uff94\uff8f\uff80\uff9e^\uff80\uff9b\uff73=\u5c71\u7530^\u592a\u90ce=\u3084\u307e\u3060^\u305f\u308d\u3046'  # noqa
         got = ds[0x32, 0x1064][0].PatientName
```

Comment:
oh I'm so glad you showed me this! I was messing around with different thing (and didn't find a solution that I liked much) and this is so much more elegant.